posthog/queries/sessions/test/test_session_recording.py

```diff
@@ -32,7 +32,7 @@ def test_query_run(self):
 
         def test_query_run_with_no_such_session(self):
             session = session_recording().run(team=self.team, session_recording_id="xxx")
-            self.assertEqual(session, {"snapshots": [], "person": None})
+            self.assertEqual(session, {"snapshots": [], "person": None, "start_time": None})
```

Comment:
Think it's worth adding the positive test case for this (i.e. `start_time` is properly set)

Reply:
Done!